validation-processor/src/main/java/io/micronaut/validation/visitor/ValidationVisitor.java

```diff
@@ -53,7 +53,7 @@ public class ValidationVisitor implements TypeElementVisitor<Object, Object> {
 
     @Override
     public Set<String> getSupportedAnnotationNames() {
-        return Set.of(ANN_CONSTRAINT, ANN_VALID);
+        return Set.of("jakarta.validation.*");
```

Comment:
please, delete unused `ANN_VALID` and `ANN_CONSTRAINT` constants.

Reply:
they are not unused